tests/test_functions.py

```diff
@@ -114,6 +115,24 @@ def test_extract_hashbits_info():
             print >>sys.stderr, '...failed to remove {fn}'.format(fn)
 
 
+def test_check_file_status_kfile():
+    fn = utils.get_temp_filename('thisfiledoesnotexist')
+    try:
+        check_file_status(fn, False)
+        assert True
```

Comment:
@ctb could you remind me of the idea behind the if/if not statement to be put here you mentioned today?